### ISSUE TITLE
[I] "Note was modified externally!" error

### ISSUE THREAD

**Issue by olegbc1** [pbek/QOwnNotes#2229]
#### Expected behavior

Creating a new note and editing it automatically saves it to my nextcloud instance, without errors about external modifications.

#### Actual behavior

Whenever typing up a note (any note) I periodically get an error saying that note was modified externally. Preview shows that markup symbols were added. Looks like nextcloud is adding some markup when the note is saved.

<img width="1112" alt="QOwnNotes_b1mpwdxOdf" src="https://user-images.githubusercontent.com/9222334/132821385-4d14921c-82ad-4b8d-a772-d6d5e495a123.png">

#### Steps to reproduce

I am on Windows 10 (21H1 19043) and synching my notes to a snap instance of NextCloud.  I am not sure the best way to handle it.

#### Relevant log output in the Log panel

<details><summary>Expand</summary>

```
[00:58:51] [status] Stored 1 note(s) to disk
[00:59:01] [status] Stored 1 note(s) to disk
[00:59:11] [status] Stored 1 note(s) to disk
[00:59:14] [status] Notes directory was modified externally
[00:59:14] [status] Stored 1 note(s) to disk
[00:59:31] [status] Stored 1 note(s) to disk
[00:59:34] [status] Current note was modified externally
[00:59:41] [status] Stored 1 note(s) to disk
[01:00:01] [status] Loading external changes from: Note 590fc1cd-8c6c-44b8-816b-9ceebff091e1.md
```
</details>

#### Information about the application, settings and environment

<details><summary>Expand</summary>

QOwnNotes Debug Information
===========================

## General Info

**Current Date**: `Fri Sep 10 01:00:14 2021`
**Version**: `21.9.2`
**Build date**: `Sep  9 2021`
**Build number**: `879`
**Platform**: `windows`
**Operating System**: `Windows 10 Version 2009`
**Build architecture**: `x86_64`
**Current architecture**: `x86_64`
**Release**: `GitHub Actions`
**Qt Version (build)**: `5.15.2`
**Qt Version (runtime)**: `5.15.2`
**Portable mode**: `no`
**Settings path / key**: `\HKEY_CURRENT_USER\Software\PBE\QOwnNotes`
**Application database path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\QOwnNotes.sqlite`
**Application arguments**: `C:\Users\olegbc\Downloads\QOwnNotes\QOwnNotes.exe`
**Qt Debug**: `0`
**Locale (system)**: `en_US`
**Locale (interface)**: *empty*
**Primary screen resolution**: `1920x1080`
**Screen resolution(s)**: `1920x1080, 1920x1080, 1920x1080` < FYI this is wrong, my 3 screens are at 4K resolution
**Icon theme**: `breeze-dark-qownnotes`
**Notes in current note folder**: `12`
**Calendar items**: `3`
**Enabled scripts**: `8`

## Server Info

**serverUrl**: `https://bezorudko.com`
**appIsValid**: `yes`
**notesPathExists**: `yes`
**serverVersion**: `21.0.4.1`
**appVersion**: *empty*

## Spellchecking

**Enabled**: `false`
**Selected language**: *empty*
**Language codes**: *empty*
**Language names**: *empty*
**Application dictionaries path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\dicts`

## Note folders

**currentNoteFolderId**: `1`

### Note folder `default`

**id**: `1`
**isCurrent**: `yes`
**activeTagId**: `10`
**localPath**: `C:\Users\olegbc\Nextcloud\Notes`
**remotePath**: `Notes`
**cloudConnectionId**: `1`
**isShowSubfolders**: `yes`
**isUseGit**: `no`
**activeNoteSubFolder name**: *empty*
**database file**: `C:\Users\olegbc\Nextcloud\Notes\notes.sqlite`

### Cloud connection `catcloud`

**id**: `1`
**isCurrent**: `yes`
**serverUrl**: `https://bezorudko.com`
**username**: `Oleg`

### Script `@tag tagging in note text (experimental)`

**id**: `1`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\in-note-text-tagging\in-note-text-tagging.qml`
**variablesJson**: 
```
{
    "putToBeginning": false,
    "tagMarker": "#"
}
```
**identifier**: `in-note-text-tagging`
**version**: `0.2.0`
**minAppVersion**: `20.6.0`

### Script `Meeting note`

**id**: `2`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\meeting-note\meeting-note.qml`
**variablesJson**: 
```
{
    "defaultTags": "Meeting",
    "headlinePrefix": "Meeting ",
    "noteBodyTemplate": "*Purpose:*\n\n*Attendees:*\n\n*Notes:*\n\n"
}
```
**identifier**: `meeting-note`
**version**: `1.4.0`
**minAppVersion**: `20.4.16`

### Script `Journal entry`

**id**: `3`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\journal-entry\journal-entry.qml`
**variablesJson**: 
```
{
    "defaultTags": "journal",
    "noteBodyTemplate": "",
    "singleJournalPerDay": true
}
```
**identifier**: `journal-entry`
**version**: `1.5.1`
**minAppVersion**: `17.11.6`

### Script `Insert Table of Contents (TOC)`

**id**: `4`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\insert-toc\insert-toc.qml`
**variablesJson**: 
```
{
    "tocLinks": true,
    "tocTitle": "Table of Contents"
}
```
**identifier**: `insert-toc`
**version**: `0.0.3`
**minAppVersion**: `17.06.2`

### Script `List maker`

**id**: `5`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\list-maker\list-maker.qml`
**variablesJson**: 
```
{
    "setLetters": "abcdefghijklmnopqrstuvwxyz",
    "setMarkers": "- • ‣"
}
```
**identifier**: `list-maker`
**version**: `0.0.2`
**minAppVersion**: `17.07.8`

### Script `Toggle checkbox`

**id**: `7`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\toggle-checkbox\toggle-checkbox.qml`
**variablesJson**: 
```
{
    "multipleLinesMethod": "cycleIndividually"
}
```
**identifier**: `toggle-checkbox`
**version**: `1.0.0`
**minAppVersion**: `19.3.2`

### Script `Insert text button`

**id**: `8`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\insert-text\insert-text.qml`
**variablesJson**: 
```
{
    "buttonIcon": "insert-text.svg",
    "buttonName": "Insert comment",
    "menuName": "Insert comment",
    "text": "[comment]: # ()"
}
```
**identifier**: `insert-text`
**version**: `0.0.1`
**minAppVersion**: `17.05.8`

### Script `Add Multiple tags`

**id**: `9`
**path**: `C:\Users\olegbc\AppData\Roaming\PBE\QOwnNotes\scripts\add-multiple-tags\add-multiple-tags.qml`
**variablesJson**: *empty*
**identifier**: `add-multiple-tags`
**version**: `0.0.1`
**minAppVersion**: `17.06.2`

## Settings

**ActiveNoteHistoryItem** (NoteHistoryItem): `<binary data>`
**Debug/fakeOldVersionNumber** (QString): `false`
**Debug/fileLogging** (QString): `false`
**DistractionFreeMode/editorWidthMode** (int): `2`
**Editor/CurrentSchemaKey** (QString): `EditorColorSchema-cdbf28fc-1ddc-4d13-bb21-6a4043316a2f`
**Editor/autoBracketClosing** (QString): `true`
**Editor/autoBracketRemoval** (QString): `true`
**Editor/disableCursorBlinking** (QString): `false`
**Editor/editorWidthInDFMOnly** (QString): `true`
**Editor/highlightCurrentLine** (QString): `true`
**Editor/indentSize** (int): `4`
**Editor/removeTrailingSpaces** (QString): `false`
**Editor/showLineNumbers** (QString): `false`
**Editor/useTabIndent** (QString): `false`
**Editor/vimMode** (QString): `false`
**IssueAssistantDialog/geometry** (QByteArray): `<binary data>`
**LastUpdateCheck** (QDateTime): `2021-09-10T00:46:21.790`
**LogWidget/criticalLog** (QString): `true`
**LogWidget/debugLog** (QString): `true`
**LogWidget/fatalLog** (QString): `true`
**LogWidget/infoLog** (QString): `true`
**LogWidget/scriptingLog** (QString): `true`
**LogWidget/statusLog** (QString): `true`
**LogWidget/warningLog** (QString): `true`
**MainWindow/geometry** (QByteArray): `<binary data>`
**MainWindow/mainToolBar.iconSize** (int): `24`
**MainWindow/menuBarGeometry** (QByteArray): `<binary data>`
**MainWindow/noteSubFolderTreeWidgetExpandState-1** (QStringList): `Current Projects`
**MainWindow/noteTextEdit.code.font** (QString): `Consolas,11,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextEdit.font** (QString): `Bookerly,12,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextView.code.font** (QString): `Courier New,9,-1,2,50,0,0,0,0,0`
**MainWindow/noteTextView.font** (QString): `MS Shell Dlg 2,6.5,-1,5,50,0,0,0,0,0`
**MainWindow/noteTextView.ignoreCodeFontSize** (QString): `true`
**MainWindow/noteTextView.rtl** (QString): `false`
**MainWindow/noteTextView.underline** (QString): `true`
**MainWindow/noteTextView.useEditorStyles** (QString): `true`
**MainWindow/noteTextView.useInternalExportStyling** (QString): `true`
**MainWindow/showNotesFromAllNoteSubFolders** (QString): `false`
**MainWindow/tagTreeWidgetExpandState-1** (QStringList): `7`
**MessageBoxOverride/remove-tags** (int): `16384`
**MessageBoxOverride/script-install-successful** (int): `1024`
**NoteDiffDialog/geometry** (QByteArray): `<binary data>`
**NoteFolder-1/NoteTabNameList** (QStringList): `Note 2021-09-10T00.44.50`
**NoteFolder-1/NoteTabStickinessList** (QStringList): *empty*
**NoteFolder-1/NoteTabSubFolderPathDataList** (QStringList): *empty*
**NoteFolder-1/allowDifferentNoteFileName** (QString): `true`
**NoteHistory-1** (QVariantList): `<variant list with 14 item(s)>`
**NoteHistoryCurrentIndex-1** (int): `13`
**PiwikClientId** (QString): `<hidden>`
**ScriptRepositoryDialog/geometry** (QByteArray): `<binary data>`
**ScriptRepositoryDialog/mainSplitterState** (QByteArray): `<binary data>`
**SearchEngineId** (int): `0`
**SettingsDialog/geometry** (QByteArray): `<binary data>`
**SettingsDialog/mainSplitterState** (QByteArray): `<binary data>`
**ShareDialog/geometry** (QByteArray): `<binary data>`
**ShowSystemTray** (QString): `true`
**StartHidden** (QString): `true`
**TagAddDialog/geometry** (QByteArray): `<binary data>`
**TodoDialog/geometry** (QByteArray): `<binary data>`
**TodoDialog/mainSplitterState** (QByteArray): `<binary data>`
**TodoDialog/showCompletedItems** (int): `0`
**TodoDialog/showDueTodayItemsOnly** (int): `0`
**TodoDialog/todoListSelectorSelectedItem** (QString): `Personal`
**UpdateDialog/geometry** (QByteArray): `<binary data>`
**VersionDialog/geometry** (QByteArray): `<binary data>`
**WelcomeDialog/geometry** (QByteArray): `<binary data>`
**acceptAllExternalModifications** (QString): `false`
**allowNoteEditing** (QString): `true`
**allowOnlyOneAppInstance** (QString): `true`
**appMetrics/disableAppHeartbeat** (QString): `false`
**appMetrics/disableTracking** (QString): `false`
**appMetrics/notificationShown** (QString): `true`
**automaticNoteFolderDatabaseClosing** (QString): `false`
**checkSpelling** (QString): `false`
**closeTodoListAfterSave** (QString): `false`
**cryptoKey** (qlonglong): `<hidden>`
**currentNoteFolderId** (int): `1`
**currentWorkspace** (QString): `5FyKDYTpKFy7`
**cursorWidth** (int): `1`
**customNoteFileExtensionList** (QStringList): *empty*
**darkMode** (QString): `true`
**darkModeColors** (QString): `true`
**darkModeIconTheme** (QString): `true`
**darkModeTrayIcon** (QString): `true`
**defaultNoteFileExtension** (QString): `md`
**disableAutomaticUpdateDialog** (QString): `true`
**disableSavedSearchesAutoCompletion** (QString): `false`
**dockWasInitializedOnce** (QString): `true`
**enableNoteTree** (QString): `true`
**enableSocketServer** (QString): `true`
**enableWebAppSupport** (QString): `false`
**externalEditorPath** (QString): *empty*
**fullyHighlightedBlockquotes** (QString): `false`
**gitCommitInterval** (int): `30`
**gitExecutablePath** (QString): *empty*
**gitLogCommand** (QString): *empty*
**guiFirstRunInit** (QString): `true`
**ignoreAllExternalModifications** (QString): `false`
**ignoreAllExternalNoteFolderChanges** (QString): `false`
**ignoreNoteSubFolders** (QString): `^\.`
**imageScaleDown** (QString): `true`
**imageScaleDownMaximumHeight** (int): `800`
**imageScaleDownMaximumWidth** (int): `800`
**initialLayoutIdentifier** (QString): `minimal`
**insertTimeFormat** (QString): *empty*
**interfaceFontSize** (int): `13`
**interfaceLanguage** (QString): *empty*
**internalIconTheme** (QString): `false`
**itemHeight** (int): `24`
**legacyLinking** (QString): `false`
**localTrash/autoCleanupDays** (int): `30`
**localTrash/autoCleanupEnabled** (QString): `true`
**localTrash/supportEnabled** (QString): `true`
**markdownHighlightingEnabled** (QString): `true`
**navigationPanelHideSearch** (QString): `false`
**networking/ignoreSSLErrors** (QString): `true`
**networking/proxyType** (int): `2`
**newNoteAskHeadline** (QString): `false`
**noteEditIsCentralWidget** (QString): `true`
**noteListPreview** (QString): `false`
**noteSaveIntervalTime** (int): `10`
**noteSubfoldersPanelDisplayAsFullTree** (QString): `true`
**noteSubfoldersPanelHideSearch** (QString): `true`
**noteSubfoldersPanelOrder** (int): `0`
**noteSubfoldersPanelShowFullPath** (QString): `false`
**noteSubfoldersPanelShowNotesRecursively** (QString): `true`
**noteSubfoldersPanelShowRootFolderName** (QString): `true`
**noteSubfoldersPanelSort** (int): `0`
**notesPanelOrder** (int): `0`
**notesPanelSort** (int): `1`
**notesPath** (QString): `C:/Users/olegbc/Nextcloud/Notes`
**notifyAllExternalModifications** (QString): `true`
**overrideInterfaceFontSize** (QString): `true`
**ownCloud/supportEnabled** (QString): `true`
**ownCloud/todoCalendarBackend** (int): `3`
**ownCloud/todoCalendarCalDAVPassword** (QString): `<hidden>`
**ownCloud/todoCalendarCalDAVServerUrl** (QString): *empty*
**ownCloud/todoCalendarCalDAVUsername** (QString): *empty*
**ownCloud/todoCalendarCloudConnectionId** (int): `1`
**ownCloud/todoCalendarDisplayNameList** (QStringList): *empty*
**ownCloud/todoCalendarEnabledList** (QStringList): *empty*
**ownCloud/todoCalendarEnabledUrlList** (QStringList): *empty*
**ownCloud/todoCalendarUrlList** (QStringList): *empty*
**ownCloudInfo/appIsValid** (QString): `true`
**ownCloudInfo/connectionErrorMessage** (QString): `Unknown error`
**ownCloudInfo/notesPathExistsText** (QString): `yes`
**ownCloudInfo/serverVersion** (QString): `21.0.4.1`
**previousWorkspace** (QString): `initial`
**recentNoteFolders** (QStringList): `C:/Users/olegbc/Dropbox/Oleg/Notes`
**restoreCursorPosition** (QString): `true`
**restoreLastNoteAtStartup** (QString): `true`
**restoreNoteTabs** (QString): `true`
**savedSearches/noteFolder-1** (QStringList): `Note 590fc1cd-8c6c-44b8-816b-9ceebff091e1, LENS, Note 2021-09-10T00.44.50, Note 2021-09-10T00.36.47, Note 2021-09-10T00.05.41, Note 6babc323-128c-4a75-a5ec-c15c6946bd1e, Note 2021-09-09T23.59.10, Note b0dc1729-344e-4f69-a273-edee419c7927, Note 2021-09-09T23.34.02, Note 2021-09-09T22.30.49, Note 2021-09-09T21.13.47`
**showMatches** (QString): `true`
**showMenuBar** (QString): `true`
**showStatusBar** (QString): `true`
**systemIconTheme** (QString): `false`
**tagWasAddedToNote** (QString): `true`
**taggingShowNotesRecursively** (QString): `true`
**tagsPanelHideNoteCount** (QString): `false`
**tagsPanelHideSearch** (QString): `true`
**tagsPanelOrder** (int): `0`
**tagsPanelSort** (int): `0`
**todoCalendarSupport** (QString): `true`
**toolbar/1/items** (QStringList): `action_New_note, action_Find_note, action_Remove_note, action_Open_note_in_external_editor, actionShow_local_trash, actionAllow_note_editing, , action_Back_in_note_history, action_Forward_in_note_history, , action_Find_text_in_note, actionReplace_in_current_note, , actionShow_versions, actionShow_trash, actionShare_note, , actionShow_Todo_List`
**toolbar/1/name** (QString): `mainToolBar`
**toolbar/1/title** (QString): `main toolbar`
**toolbar/2/items** (QStringList): `actionFormat_text_bold, actionFormat_text_italic, actionStrike_out_text, actionInsert_code_block, actionInsert_block_quote`
**toolbar/2/name** (QString): `formattingToolbar`
**toolbar/2/title** (QString): `formatting toolbar`
**toolbar/3/items** (QStringList): `actionInsert_text_link, actionInsert_image, actionInsert_current_time, actionInsert_attachment, actionInsert_table`
**toolbar/3/name** (QString): `insertingToolbar`
**toolbar/3/title** (QString): `inserting toolbar`
**toolbar/4/items** (QStringList): *empty*
**toolbar/4/name** (QString): `encryptionToolbar`
**toolbar/4/title** (QString): `encryption toolbar`
**toolbar/5/items** (QStringList): `actionWorkspaceComboBox, actionStore_as_new_workspace, actionRemove_current_workspace, actionRename_current_workspace, actionSwitch_to_previous_workspace, actionUnlock_panels, , actionToggle_distraction_free_mode, action_Increase_note_text_size, action_Decrease_note_text_size, action_Reset_note_text_size`
**toolbar/5/name** (QString): `windowToolbar`
**toolbar/5/title** (QString): `window toolbar`
**toolbar/6/items** (QStringList): `action_Quit`
**toolbar/6/name** (QString): `quitToolbar`
**toolbar/6/title** (QString): `quit toolbar`
**toolbar/size** (int): `6`
**useNoteFolderButtons** (QString): `false`
**useUNIXNewline** (QString): `true`
**versionSplitterSizes** (QByteArray): `<binary data>`
**webAppClientService/serverUrl** (QString): `wss://app.qownnotes.org`
**webAppClientService/token** (QString): `<hidden>`
**webSocketServerService/bookmarksNoteName** (QString): `Bookmarks`
**webSocketServerService/bookmarksTag** (QString): `bookmarks`
**webSocketServerService/port** (int): `22222`
**webSocketServerService/token** (QString): `<hidden>`
**workspace-5FyKDYTpKFy7/name** (QString): `Minimal`
**workspace-5FyKDYTpKFy7/noteSubFolderDockWidgetVisible** (QString): `true`
**workspace-5FyKDYTpKFy7/windowState** (QByteArray): `<binary data>`
**workspace-initial/name** (QString): `Full`
**workspace-initial/noteSubFolderDockWidgetVisible** (QString): `true`
**workspace-initial/windowState** (QByteArray): `<binary data>`
**workspaces** (QStringList): `initial, 5FyKDYTpKFy7`

## System environment

**ALLUSERSPROFILE**: `C:\ProgramData`
**APPDATA**: `C:\Users\olegbc\AppData\Roaming`
**CommonProgramFiles**: `C:\Program Files\Common Files`
**CommonProgramFiles(x86)**: `C:\Program Files (x86)\Common Files`
**CommonProgramW6432**: `C:\Program Files\Common Files`
**COMPUTERNAME**: `REMOVED`
**ComSpec**: `C:\WINDOWS\system32\cmd.exe`
**DriverData**: `C:\Windows\System32\Drivers\DriverData`
**HOMEDRIVE**: `C:`
**HOMEPATH**: `\Users\olegbc`
**LOCALAPPDATA**: `C:\Users\olegbc\AppData\Local`
**LOGONSERVER**: `\\DC-USW2B-11`
**NUMBER_OF_PROCESSORS**: `8`
**OneDrive**: `C:\Users\olegbc\OneDrive`
**OS**: `Windows_NT`
**Path**: `C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Program Files\itacsvc;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files (x86)\Quarantine\;C:\Program Files (x86)\ACMEGuardian\;C:\Program Files\dotnet\;C:\Program Files\Git\cmd;C:\Program Files\PowerShell\7\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\PuTTY\;C:\Users\olegbc\AppData\Local\Microsoft\WindowsApps;C:\Program Files (x86)\Nmap;C:\Users\olegbc\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\olegbc\.dotnet\tools`
**PATHEXT**: `.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC`
**POWERSHELL_DISTRIBUTION_CHANNEL**: `MSI:Windows 10 Enterprise`
**PROCESSOR_ARCHITECTURE**: `AMD64`
**PROCESSOR_IDENTIFIER**: `Intel64 Family 6 Model 140 Stepping 1, GenuineIntel`
**PROCESSOR_LEVEL**: `6`
**PROCESSOR_REVISION**: `8c01`
**ProgramData**: `C:\ProgramData`
**ProgramFiles**: `C:\Program Files`
**ProgramFiles(x86)**: `C:\Program Files (x86)`
**ProgramW6432**: `C:\Program Files`
**PSModulePath**: `C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules`
**PUBLIC**: `C:\Users\Public`
**SESSIONNAME**: `Console`
**SystemDrive**: `C:`
**SystemRoot**: `C:\WINDOWS`
**TEMP**: `C:\Users\olegbc\AppData\Local\Temp`
**TMP**: `C:\Users\olegbc\AppData\Local\Temp`
**UATDATA**: `C:\WINDOWS\CCM\UATData\D9F8C395-CAB8-491d-B8AC-179A1FE1BE77`
**USERDNSDOMAIN**: `REMOVED`
**USERDOMAIN**: `REMOVED`
**USERDOMAIN_ROAMINGPROFILE**: `REMOVED`
**USERNAME**: `olegbc`
**USERPROFILE**: `C:\Users\olegbc`
**windir**: `C:\WINDOWS`
**ZES_ENABLE_SYSMAN**: `1`
</details>

**Comment by pbek** [comment]
Sounds like those: https://github.com/pbek/QOwnNotes/issues?q=Note+was+modified+externally

**Comment by pbek** [comment]
TL;DR: If you can't find the culprit who is writing into your notes you can turn off the check for external modifications in the *General settings*.

**Comment by olegbc1** [comment]
Thanks! It's a new nextcloud server with only one user enabled so nobody could be writing in my notes, but it looks like nextcloud is adding some markup. Could it be triggering the conflict dialog?

**Comment by pbek** [comment]
I use Nextcloud myself and only have a similar issue now and then under macOS (nothing under Windows or Linux), where maybe spotlight is doing something.

Do you maybe have a 2nd computer with QOwnNotes active and a synclient from there does some interfering?

Or some other application does something to the note file before QON stores it again while you are typing.

**Comment by olegbc1** [comment]
> Do you maybe have a 2nd computer with QOwnNotes active and a synclient from there does some interfering?

I've only just started setting everything up and actually discovered NextCloud while looking for a good application to manage all my notetaking and coming across QOwnNotes. So many of such apps are based on electron and tend to lag pretty hard when I am compiling or even just sitting on a video call... 

All of this is to really say that I only currently have one client and one user so client are probably not the culprit. I did install everything kind of backwards though. First QOwnNotes, then NextCloud instance on Lightsail, and only then the NextCloud client on my PC. Could there be something out of whack between NextCloud client sync and QON API app on NextCloud? I'm not sure how to test but I am seeing the error consistently and feels like I should not just silence it.

**Comment by pbek** [comment]
> First QOwnNotes, then NextCloud instance on Lightsail, and only then the NextCloud client on my PC. 

That shouldn't cause any issues...

> Could there be something out of whack between NextCloud client sync and QON API app on NextCloud?

No, QON doens't do any file syncing or other Nextcloud file operations.

Maybe a virus scanner or search index services is doing something? I never had any issue with NC & QON on Windows 10.

Do you have the same issue when running QOwnNotes in a new session (see [cli-parameters](https://www.qownnotes.org/getting-started/cli-parameters.html))?

**Comment by netguerrilla12** [comment]
I set the setting in General Settings to ignore the external changes. I also have unchecked the check box for notifications about external changes. I'm still seeing notifications in the lower left of QON, "Current note was modified externally." First, I've never modified any notes externally. Second, if I turned off the notifications, why do I still get them? It's not a huge deal but it's a bit annoying.

**Comment by pbek** [comment]
You don't get the diff dialog in the editor.

**Comment by netguerrilla12** [comment]
I'm not getting the big long message anymore but I still get a notification every so often. I'll post a screen shot here the next time it happens.

**Comment by netguerrilla12** [comment]
It just appeared again but it disappeared too quick for me to get a screen shot of it.
